### PR TITLE
Use getClientId instead of getName for getting clients' name

### PR DESCRIPTION
**Pull request by danifr**
Hi!

When looking at the logs messages:

> Jan 24 09:53:41 auth.bbp.epfl.ch keycloak[2359967]: 2023-01-24 09:53:41,858 WARN  [de.sventorben.keycloak.authorization.client.access.role.ClientRoleBasedAccessProvider] (executor-thread-8) Access for user 'danielfr' to client '' in realm 'BBP' is denied. User does not have client role 'restricted-access' on client with id 'b5c85eb7-9a94-4058-b483-a2e7f29f995e'.

The client name seems to be empty.

The current version of the authenticator uses `getName()` to get the clients' name, but this attribute is optional in Keycloak.
I'd suggest changing it by `getClientId()`  (https://github.com/keycloak/keycloak/blob/main/server-spi/src/main/java/org/keycloak/models/ClientModel.java#L112) that returns the mandatory "client ID" attribute as defined by the user.

### CODE REVIEW

**Comment by sventorben** [comment]
Thanks @danifr 
Happy to merge once the checks are green.

File: src/main/java/de/sventorben/keycloak/authorization/client/access/role/ClientRoleBasedAccessProvider.java [hunk 1]
```diff
@@ -30,10 +30,10 @@ public boolean isPermitted(ClientModel client, UserModel user) {
         if (permitted) {
             LOG.debugf(
                 "Access for user '%s' to client '%s' in realm '%s' granted.",
-                user.getUsername(), client.getName(), client.getRealm().getName());
+                user.getUsername(), client.getClientId() client.getRealm().getName());
```
Comment (sventorben) [inline src/main/java/de/sventorben/keycloak/authorization/client/access/role/ClientRoleBasedAccessProvider.java:33]: missing a comma here

**Comment by sventorben** [comment]
Thank you!

**Comment by danifr** [comment]
Hi Sven-Torben,

do you think you can release a new version of this package (v.20.0.X or the version number you consider) including this patch and the other changes that landed in `main`?

Thank you very much!

**Comment by sventorben** [comment]
Sure! It's 20.0.1
https://github.com/sventorben/keycloak-restrict-client-auth/releases/tag/v20.0.1

**Pull request merged**